UM-USElab/gradecraft-development
# Show a message in the Grading tab and the quick grade page if there are no groups for a group assignment

### Status
**READY**

### Description
* When grading a group assignment with no groups as an instructor a table with no entries is shown under the "Grading" tab and in the quick grade page. 
* Now a message is displayed if the group assignment does not have any groups

### Migrations
NO

### Steps to Test or Reproduce
1. As an instructor create a group assignment without any groups and view the "Grading" tab and the quick grade page. A message instructing the user to  add groups for the assignment to see grades is displayed.

### Impacted Areas in Application
* Viewing grades for a group assignment as an instructor (i.e. views/assignments/group/_group_show.haml)
** Viewing the quick grade page for a group assignment as an instructor (i.e. views/assignments/groups/grades/mass_edit.html.haml)

======================
Closes #4336
